docs/polkadot/Polkadot-Host-Spec.md

```diff
@@ -15,4 +15,5 @@ presented in a more readable way in preceding documents.
 
 TODO: move to publications
 
-<iframe src="../_static/pdfview/viewer.html?file=../pdf/polkadot-host-spec.pdf" width="100%" height="650em"></iframe>
+<iframe src="../_static/pdfview/viewer.html?file=https://w3f.github.io/polkadot-spec/spec/host/latest.pdf" width="100%" height="650em"></iframe>
```

Comment:
Did you test that this actually works? I seem to remember some cross-domain restriction or maybe it was just because I was using file:///

Reply:
Ok, damn. It is not that easy. Will get back to you on this.
```
Message: file origin does not match viewer's
```

Reply:
This seems to be more a limit of the viewer. I will just switch to a direct embedding, which does not seem to cause that issue for me:
```
<iframe src="https://w3f.github.io/polkadot-spec/spec/host/latest.pdf" width="100%" height="650em"></iframe>
```

Reply:
It might be possible to configure the view to allow some whitelisted other domains, I'll have a look into it..

Reply:
(the error messages comes from the viewer's own js, you can see it in the git repo, so it is probably possible)

Reply:
@FlorianFranzen try this

```diff
diff --git a/docs/_static/pdfview/viewer.js b/docs/_static/pdfview/viewer.js
index 4c65e33..2aaaa9a 100644
--- a/docs/_static/pdfview/viewer.js
+++ b/docs/_static/pdfview/viewer.js
@@ -1546,7 +1546,7 @@ var PDFViewerApplication = {
 };
 var validateFileURL = void 0;
 {
-  var HOSTED_VIEWER_ORIGINS = ['null', 'http://mozilla.github.io', 'https://mozilla.github.io'];
+  var HOSTED_VIEWER_ORIGINS = ['null', 'http://mozilla.github.io', 'https://mozilla.github.io', 'https://w3f.github.io'];
   validateFileURL = function validateFileURL(file) {
     if (file === undefined) {
       return;
```

Reply:
What is the benefit of hosting our own viewer and not relying on each browsers own PDF viewer?

Reply:
I actually don't know, "it was like that when I got here". We can chat a bit about it in the sync meeting tomorrow?

Reply:
For reference, it was added in #14. Both of them (federico, peter) have left W3F now...

Reply:
Ok, it seems I am spoiled by Firefox, which just has pdf.js build in. I update the local version instead and replace the mozilla with the w3f github pages subdomain (in accordance with mozilla/pdf.js#6916).